### ISSUE TITLE
Presto does not find tables with camelCase

### ISSUE THREAD

**Issue by mizunno** [prestosql/presto#2394]
Hi everyone!

I am trying to select data from a table with its name written in camelCase format, something like:

`SELECT * FROM catalog.schema.tableName`

But Presto returns:

`Table catalog.schema.tableName does not exist`

If I run `SHOW TABLES FROM catalog.schema` Presto returns all the tables in lowercase. I've tried to put the table name between quotation marks ("") but the problem still remains.

Is there any work around for this? Either in the Presto side or in the SQL side.

**Note:** I'm using PrestoDB 0.227 in EMR cluster.
**Note 2:** Also tried this in PrestoDB 0.229.

**Comment by Praveen2112** [comment]
hi!! Presto transforms all the identifiers - tables, schema, columns to lower case. There is a on-going PR (#2350) which solves this issue. For JDBC based connectors we can enable case sensitive matching of tableNames by setting this property `case-insensitive-name-matching` as true

**Comment by findepi** [comment]
For the reference, I implemented `case-insensitive-name-matching` in Presto 309, but I think PrestoDB backported this to their fork, so it I guess it's available in .227 as well.
Anyway,as you're using JDBC connectors, I recommend that you try Presto 327 https://prestosql.io/download.html -- you're missing A TON of improvements that we developed within JDBC connectors (and everywhere else).

Let me close the issue, as I think your question was already answered.
We can continue discussion here, or on [Presto community slack](https://prestosql.io/slack.html).
See you there!
